### PR TITLE
Add New roles and update EMM and SDR inbound

### PR DESCRIPTION
**Pull request by gabrielamatias**
Add new roles:
- NBD Representative
- Market analyst intern
- Customer Support: Germany
- SDR: Germany

Update:
- SDR: Inbound
- European Marketing Manager

@jesssummerfield , @jutta-frieden , @jmrobins : please review.

### CODE REVIEW

**Comment by gabrielamatias** [comment]
@harrison , @waltfy : can you help? I can't understand from circle CI what's wrong.

**Comment by harrison** [comment]
@gabrielamatias might have fixed the tests, something really weird was going on with links that had an empty `href` attribute. Instead of opening the dropdown it would reload the page instead.

**Comment by harrison** [comment]
Pretty sure it's not your fault btw!

**Comment by gabrielamatias** [comment]
Thank you!!!!

File: app/pages/about/jobs/positions/market-analyst-internship.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            The motivation and flexibility to work well in a rapidly changing, high-growth environment.
+          </li>
+          <li>
+            Superb verbal and written communication skills.
+          </li>
+        </ul>
+        <h4 className='u-text-xs u-margin-Bxs u-margin-Tm'>The perks</h4>
+        <ul className='list u-margin-Tm'>
+          <li>
+            Competitive salary, commission and options package.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/market-analyst-internship.js]: @gabrielamatias is the analyst position commissioned? Also they won't have options.

File: app/pages/about/jobs/positions/market-analyst-internship.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            A numerate individual that relishes the process of analysing data to extract insights.
+          </li>
+          <li>
+            A passion for technology, the startup community, and supporting other businesses as they grow.
+          </li>
+          <li>
+            The motivation and flexibility to work well in a rapidly changing, high-growth environment.
+          </li>
+          <li>
+            Superb verbal and written communication skills.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/market-analyst-internship.js]: @gabrielamatias can you change the 'what we're looking for' section so it's more like the other ads e.g., required skills in bullets and then a couple of sentences on the ideal candidate having a passion for technology/startup community etc

File: app/pages/about/jobs/positions/new-business-development-representative.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+          before passing qualified opportunities on to our Account Executives.
+        </p>
+        <p className='para'>
+          We generate new opportunities a little differently at GoCardless. There will be no cold calling and instead a more
+          consultative and targeted approach is taken. The NBD team is focussed on developing large enterprise opportunities
+          for GoCardless so a deep understanding of our target markets and prospects is essential to succeed. We use a range
+          of tools within the position and there are a number of opportunities to attend industry focussed events.
+        </p>
+        <p className='para'>
+          Training for the position will involve a 4 week structured program split across both our NBD team and our
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/new-business-development-representative.js]: @gabrielamatias can you change to four not 4

File: app/pages/about/jobs/positions/new-business-development-representative.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+          Sales Development team. The program will give you an in depth overview of the product, how we do sales at
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/new-business-development-representative.js]: @gabrielamatias change to "split across our NBD and Sales Development teams"

File: app/pages/about/jobs/positions/sales-development-representative-germany.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+      <JobsPage>
+        <h3 className='u-text-heading-light u-text-m u-color-dark-gray'>Sales Development Representative: Germany</h3>
+        <p className='para'>
+          Drive GoCardless’s global growth.
+        </p>
+        <p className='para'>
+          At GoCardless we are looking to scale from processing $1 billion a year to over $10 billion worldwide.
+          Our <a href='https://gocardless.com/blog/a-new-payment-network-for-the-internet/' target='_blank'> vision </a>
+          is to create a global payments network, making payments simpler on
+          the internet no matter what country you're in. We launched in Germany in 2015 and have added noteworthy customers and partners, such as
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-germany.js]: @gabrielamatias change to "...have added several notable customers..."

File: app/pages/about/jobs/positions/sales-development-representative-germany.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        <h4 className='u-text-xs u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+          The sales team at GoCardless combines deep product expertise with a commitment to helping new and existing businesses
+          succeed across a range of industries. As the first sales development representative for Germany, you'll work to uncover and define
+          our best growth opportunities, including new customers, verticals and sales strategies. You'll manage inbound leads as
+          the first point of contact for many new and growing businesses interested in us, and you’ll run outbound campaigns to reach
+          new merchants. Along the way, you'll work closely with business and product leaders across the company to help accelerate our growth.
+        </p>
+        <p className='para'>
+          Training for the position will involve a <a href='https://gocardless.com/blog/how-we-train-salespeople/' target='_blank'> 4 week structured program </a>
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-germany.js]: @gabrielamatias 'four' not '4'

File: app/pages/about/jobs/positions/sales-development-representative-germany.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+          where you’ll develop a detailed understanding of the product and where we fit within the payments industry. We’ll also invest time in
+          developing your sales skills through one-to-one sessions, mock calls and demos with the wider sales team.
+        </p>
+        <h4 className='u-text-xs u-margin-Bxs u-margin-Tm'>What we're looking for</h4>
+        <ul className='list u-margin-Tm'>
+          <li>
+            You have experience within a sales environment, a top consultancy,
+            or are a recent graduate with commercial or technical background.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-germany.js]: @gabrielamatias change to "...recent graduate with a commercial or technical background" and can you check with RBS whether we can specify recent graduate (same for market analyst intern position)?

File: app/pages/about/jobs/positions/sales-development-representative-germany.js [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+          </li>
+          <li>
+            Bi-weekly and monthly all team lunches and evenings out.
+          </li>
+          <li>
+            Stocked kitchen and unlimited (great) coffee!
+          </li>
+        </ul>
+        <p className='para'>
+          If you’re unsure, please apply &ndash; we’d love to hear from you. This role will be based in London.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-germany.js]: @gabrielamatias can this be the same as other ads i.e., "However our team comes from a variety of backgrounds and we welcome diversity – if you’re unsure, please apply." Also in the German support ad, the bit on the role being based in London is on a different line. Don't have a preference but can we keep the formatting consistent?

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 9]
```diff
@@ -13,61 +13,61 @@ export default class SalesDevelopmentRepresentativeInbound extends React.Compone
...
         <p className='para'>
-          At GoCardless we are looking to scale from processing $1 billion a year to over $10 billion.
+          At GoCardless we are looking to scale from processing $1 billion a year to over $10 billion worldwide.
           Our <a href='https://gocardless.com/blog/a-new-payment-network-for-the-internet/' target='_blank'> vision </a>
           is to create a global payments network, making payments simpler on
           the internet no matter what country you're in.
         </p>
         <p className='para'>
+          Thousands of companies now use our services, from local authorities, energy suppliers and charities to household
+          names like Thomas Cook, The Guardian and The Financial Times. Following our recent $13m Series C funding round,
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: @gabrielamatias worth linking to an article about this?

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 10]
```diff
@@ -13,61 +13,61 @@ export default class SalesDevelopmentRepresentativeInbound extends React.Compone
...
           commitment to helping new and existing businesses succeed across a
-          range of industries. As a sales development rep, you'll work to
-          uncover and define our best growth opportunities, including new
-          customers, verticals and sales strategies. You'll manage inbound
-          leads as the first point of contact for many new and growing
-          businesses interested in us, and you’ll run outbound campaigns to
-          reach new merchants. Along the way, you'll work closely with
-          business and product leaders across the company to help accelerate
-          our growth.
+          range of industries. As a Sales Development Rep, you’re the first port of call for many of our prospective customers.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: @gabrielamatias change to "you'll be the first point of call" so tense is consistent with next sentence

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 11]
```diff
@@ -13,61 +13,61 @@ export default class SalesDevelopmentRepresentativeInbound extends React.Compone
...
+          You’ll mix in-depth knowledge of the product and wider payments industry with a consultative sales approach to
+          help fit the product to the customer’s specific requirements. This is a high impact role. You’ll have the unique
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: @gabrielamatias change to "This is a high impact role, and you'll have the opportunity..."

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 12]
```diff
@@ -13,61 +13,61 @@ export default class SalesDevelopmentRepresentativeInbound extends React.Compone
...
+          opportunity to work closely with Marketing, Product and Ops and shape the way our product is used.
         </p>
         <p className='para'>
-          Training for the position will involve a 4 week structured program split across both our NBD team as
-          well as our Sales Development team. The program will give you an in depth overview of the product,
-          how we do sales at GoCardless, and a detailed understanding of how we fit into the wider payments industry.
+          Training for the position will involve a <a href='https://gocardless.com/blog/how-we-train-salespeople/' target='_blank'>  4 week structured program </a>
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: @gabrielamatias four not 4

**Comment by jesssummerfield** [comment]
@gabrielamatias done with my comments, sorry I did them in stages!

**Pull request merged**